[CI:DOCS] Add section about current differences (#7797)
containers/podman: transfer.md

**** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](./docs/podman-cp.1.md) for more information.

## Behavioural differences and pitfalls

> These commands behave different from Docker:

Reviewer: ```suggestion
These commands behave differently from the commands in Docker:
```

| Command | Description |
| :--- | :--- |

> | `podman volume create` | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) on that.|

Reviewer: ```suggestion
| `podman volume create` | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) for more information.|
```
